gulpfile.js

```diff
@@ -58,7 +58,10 @@ function bundle() {
 
 /* Compile SASS */
 gulp.task('sass', function() {
-  return gulp.src('pages/static/src/scss/main.scss')
+  return gulp.src([
+    'pages/static/src/scss/main.scss',
+    'pages/static/src/scss/admin.scss'
```

Comment:
Does this look right? Want to process both files in similar way but don't want to merge them together.

Reply:
Yup, this works!